src/main/java/uk/gov/hmcts/reform/sendletter/services/date/DateService.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.sendletter.services.date;
+
+import org.springframework.stereotype.Service;
+
+import java.time.ZonedDateTime;
+
+@Service
+public class DateService {
```

Comment:
```suggestion
public class DateCalculator {
```

Reply:
Had problems with the previous date, too